### ISSUE TITLE
Export summary and detailed reports from Reports section based on data shown

### ISSUE THREAD

**Issue by wordmaid** [AffiliateWP/AffiliateWP#2223]
Blue-sky dreaming here, but it would be amazing to be able to export a summary and detailed data directly from the report screen.

From the Reports section, for each tab (referrals/affiliates/visits/campaigns), allow the user to export a report based on the data they are viewing (using a specified date filter. This would show the current displayed summary, but also includes all referral/affiliate/visit/campaign data within that period. 😱

Or, they could choose between the summary and the detailed reports.

Am I crazy? Yes, yes, I just might be.

**Comment by pippinsplugins** [comment]
Yes this would be awesome.

**Comment by DrewAPicture** [comment]
Yeah, this kind of follows the same logic behind the WYSIWYE (what you see is what you export) concept for the list tables. Eventually, I'd likely see a lot more parity between the list tables and reports where the tables hold a tabular representation filtered data and reports show a visual one with accompanying stats. Either way way, I'd really like to look at our options for WYSIWYE.

The biggest unanswered question is not really how to actually export the data – we already know how to do that, but rather _in what format_ to export it that best reflects what you currently see?
